web_responsive/static/src/js/web_responsive.js

```diff
@@ -307,11 +311,42 @@ odoo.define('web_responsive', function(require) {
         },
     });
 
+    // Chatter Side
+    FormRenderer.include({
+
+        _updateView: function ($newContent) {
+            this._super($newContent);
+            var self = this;
+            var $chatter = this.$el.find('.oe_chatter');
+            if (this.has_sheet && $chatter.length !== 0) {
+                Rpc.query({
```

Comment:
This is not the good hook to call for this. You're asking for the setting before you render each form. A normal user session renders many forms, so we should try to save requests whenever possible, as they affect directly to the performance. You have 2 options, to avoid doing so many calls:

- Call once, cache request. Similar to what's done in #954.
- Manage this from the backend. Overwrite [template `web.webclient_bootstrap`](https://github.com/odoo/odoo/blob/dbdd742d2046f208c0057fdc60b359e5c88814ee/addons/web/views/webclient_templates.xml#L642), by adding a class such as `chatter-position-#{user.chatter_position}`.

  Then, no JS is needed at all; you can reposition it through Less directly.

IMHO I'd go for 2nd option. I think it will be faster for the user and easier to maintain. But you choose :wink: